Makefile

```diff
@@ -182,7 +193,7 @@ status-go-deps:
 	go install github.com/kevinburke/go-bindata/v4/...@v4.0.2
 	go install google.golang.org/protobuf/cmd/protoc-gen-go@v1.34.1
 
-statusgo-android: generate
+statusgo-android: generate-sds-android generate
```

Comment:
you can avoid making changes to android target now, we will deprecate this way of linking soon.
I've made the jobs in CI optional.
The new mobile app will rely on `statusgo-library` target.